src/verilog.y

```diff
@@ -1861,7 +1861,7 @@ data_typeNoRef<nodeDTypep>:		// ==IEEE: data_type, excluding class_type etc refe
 										       SYMP,VFlagChildDType(),$1); }
 	|	ySTRING					{ $$ = new AstBasicDType($1,VBasicDTypeKwd::STRING); }
 	|	yCHANDLE				{ $$ = new AstBasicDType($1,VBasicDTypeKwd::CHANDLE); }
-	|	yEVENT					{ $$ = new AstBasicDType($1,VBasicDTypeKwd::EVENTVALUE); }
+	|	yEVENT					{ $$ = new AstBasicDType($1,VBasicDTypeKwd::EVENT); v3Global.setHasEvents(); }
```

Comment:
```suggestion
	|	yEVENT					{ $$ = new AstBasicDType{$1, VBasicDTypeKwd::EVENT}; v3Global.setHasEvents(); }
```

Please cleanup spacing & constructor style when editing lines in here.  Ditto below.